model/src/w3iopomd.F90

```diff
@@ -596,9 +596,10 @@ SUBROUTINE W3IOPP ( NPT, XPT, YPT, PNAMES, IMOD, MPI_COMM_IOPP )
         ncerr = nf90_get_var(fh, v_ptifac, PTIFAC, start = (/ 1, 1/), &
           count = (/ d_wghtlen_len, d_nopts_len /))
         if (nf90_err(ncerr) .ne. 0) return
-      END IF 
+      END IF
```

Comment:
The permissions on this file have changed (see 100644->100755 where the name of the file is).  Please revert this change.

Reply:
Thanks for updating this @AliS-Noaa